app/controllers/api/v1x0.rb

```diff
@@ -1,3 +1,4 @@
+require 'services/api/v1x0'
```

Comment:
I was going to ask if we need this in the `app/services/api/v1x1.rb` file as well, but I think we're fully qualifying all of those names so I don't think we do.